client/src/components/views/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useRef, useState } from 'react';
+import { Flex, FormLabel, Icon, Input, Box, useToast } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import Head from 'next/head';
+import { GrLock } from 'react-icons/gr';
+import { Heading, Text } from '../common/Typography.jsx';
+import { FormButton } from '../common/Button.jsx';
+import { StyledForm } from '../common/Form.jsx';
+import { userResetPassword } from 'utils/api/users.js';
+
+const ValidationText = styled(Text)`
+  font-size: 10px;
+  padding-left: 12px;
+  width: 400px;
+`;
+
+const Container = styled(Box)`
+  width: 100%;
+  height: 456px;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  left: 50%;
+  align-items: center;
+  transform: translate(-50%, -50%);
+  background-color: white;
+  padding: 10px;
+  max-width: 560px;
+`;
+
+const InputContainer = styled(Flex)`
+  flex-direction: row;
+  background-color: white;
+  border: 1px solid #d5d5d5;
+  border-radius: 4px;
+  width: 100%;
+  max-width: 380px;
+  font-family: Montserrat;
+  font-size: 14px;
+`;
+
+const UnstyledInput = styled(Input)`
+  border: none;
+`;
+
+const InputIcon = styled(Icon)`
+  width: 30px;
+  height: 30px;
+  margin: 15px;
+`;
+
+const StyledFormLabel = styled(FormLabel)`
+  padding: 0 1rem;
+  margin-top: 10px;
+  margin-bottom: unset;
+  color: #9c9c9c;
+  font-size: 12px;
+`;
+
+const ResetPasswordPage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const resetPasswordRef = useRef('');
+  const confirmResetPasswordRef = useRef('');
+  const toast = useToast();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    if (resetPasswordRef.current.value !== confirmResetPasswordRef.current.value) {
+      toast({
+        title: 'ERROR',
+        description: 'The passwords does not match',
+        status: 'error',
+        duration: 2000,
+        isClosable: true,
+      });
+      setIsLoading(false);
+      return;
+    }
+
+    // min 8 characters one uppercase, one lowercase, one number, one special character
+    const regex = /^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{8,}$/;
```

Comment:
Do you think we should use a common reference to the password validation regex to be consistent?

Reply:
new file called [usePasswordValidation] created inside hooks folder, both password regex point to this file, I will push the commit